### ISSUE TITLE
Create different levels of importance for dialogs

### ISSUE THREAD

**Issue by oksks** [citizenos/citizenos-fe-old#489]
**What is the problem?**
I clicked "Send to voting" from within a topic, and got a pop-up that said "**The topic will close**, its contents can no longer be edited". 
I'd rephrase it to "Topic contents will be locked" or something similar instead of "closed", as closing something usually equals to permanent deletion, especially with the warning sign and big red font.

<img width="630" alt="Screenshot 2020-05-11 at 11 43 22" src="https://user-images.githubusercontent.com/34126353/81545269-512a1c80-9381-11ea-91cb-727ed401c57c.png">

Posted this to Slack, @kevincrepin commented:

> Kevin:
good point, this design was at that moment made for deleting a topic, but it’s been used for other “bigger decisions” as well I see. we need some different levels of importance for the dialogs

**Why is this a problem?**
It looks misleading and scary to the user, and might prevent them from sending the topic to voting.

**Possible solution.**
Create different, more friendly and supportive style pop-ups for non-terminal actions.

**Comment by kevincrepin** [comment]
We already had one other type of "attention seeking" dialog in place. I've made minor changes to these kinds of dialogs to bring them in line with our other dialogs. 

https://projects.invisionapp.com/d/main#/console/9829159/418066154/preview

Some examples of how they're visually different but it needs to be decided which are "red" and which are "blue" categories.

**Comment by loorm** [comment]
Assigned for Triage.

**Comment by loorm** [comment]
Triage 22. Est dev time 2-4 hours. Prioritizing.

**Comment by KatiVellak** [comment]
Legally reviewed, no impact.

**Comment by ilmartyrk** [comment]
@kevincrepin can you please add info circle x2 icon, svg is OK too :)

**Comment by kevincrepin** [comment]
@ilmartyrk here you go 👍 
[icon_delete_topic.svg.zip](https://github.com/citizenos/citizenos-fe/files/5785772/icon_delete_topic.svg.zip)
